app/components/benchmark-report.gjs

```diff
@@ -0,0 +1,192 @@
+import Component from "@glimmer/component";
+import { tracked } from "@glimmer/tracking";
+import { fn } from "@ember/helper";
+import { on } from "@ember/modifier";
+import { action, computed } from "@ember/object";
+import { equal, gt } from "@ember/object/computed";
+import { htmlSafe } from "@ember/template";
+import { formatNumber } from "../helpers/format-number";
+import AreaChart from "./area-chart";
+
+export default class BenchmarkReport extends Component {
+  @tracked mode = "html";
+  @equal("mode", "html") isHtmlMode;
+  @equal("mode", "text") isTextMode;
+  @gt("args.report.testGroupReports.length", 1) showGraph;
+
+  chartOptions = {
+    title: "Time (ms) (lower is better)",
+    hAxis: {
+      title: "Ember Version",
+    },
+    intervals: { style: "area" },
+    legend: "none",
+  };
+
+  @computed("args.report.testGroupReports.[]")
```

Comment:
It might be a good idea to also pull in `tracked-built-ins` for stuff like `TrackedArray` - to avoid using `computed` in glimmer components

Reply:
Thinking abou it `@computed` is not really needed.

`args.report` will be fully updated and is tracked.